### ISSUE TITLE
Address absolute/relative paths on web

### ISSUE THREAD

**Issue by RealWeeks** [ga-wdi-boston/unix-cli#15]
See https://github.com/ga-wdi-boston/unix-cli-diagnostic/issues/65  @payne-chris-r

**Comment by RealWeeks** [comment]
Moved to address previous issue

**Comment by gaand** [comment]
I'm not sure I understand.  Relative to what?

**Comment by payne-chris-r** [comment]
A link on the web is an absolute path. We don't specifically call that out in the README, but we do ask them about it in the Diagnostic. That's the entire issue.

**Comment by jrhorn424** [comment]
@ga-wdi-boston/core I believe this is out of scope, no? Besides, I'm confused by what is meant by "a link on the web is an absolute path". It isn't, nor does it need to be: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types

**Comment by RealWeeks** [comment]
I believe @jrhorn424 is correct that this is out of scope. I'm going to close, feel free to reopen if anyone wants further discussion.

**Comment by gaand** [comment]
It either needs to be addressed here or removed from the diagnostic.

**Comment by jrhorn424** [comment]
I agree that this continues to be a problem. We should make a decision. 

https://github.com/ga-wdi-boston/unix-cli-diagnostic/commit/c957ea49a428fd70e1ff1422b1090cb9176816fc#commitcomment-20461559

**Comment by tvcrawley** [comment]
The absolute/relative path question on the diagnostic has been changed for LM01 to reference the file system.
